docs/self-hosted/config-options.md

```diff
@@ -339,6 +339,17 @@ WebSockets work reliably across multiple containers of Directus.
 
 Redis is required when you run Directus load balanced across multiple containers/processes.
 
+### Data Retention
+
+| Variable              | Description                                                                                                      | Default Value |
+| --------------------- | ---------------------------------------------------------------------------------------------------------------- | ------------- |
+| `RETENTION_ENABLED`   | Whether or not to enable data retention.                                                                         | `false`       |
```

Comment:
What happens when it is set to true/false? For logs to exist at all, there needs to be some form of retention.

Reply:
Unless this means kind of the opposite - which is by enabling it, auto-deletion of retention will take place. In which case the description should be something like: 

'Whether or not to enable custom data retention settings. `false` will not delete data.'

Reply:
It is the second (enabling will have auto deletion take place)

Reply:
```suggestion
| `RETENTION_ENABLED`   | Whether or not to enable custom data retention settings. `false` will not delete data.       |
```

Reply:
I was just looking into how to enable this new feature, and was also slightly confused by the naming of the variable, as one would logically assume that by "enabling retention", I was turning ON the ability for data to be retained, rather than turning on the ability to not retain it.
Perhaps "CUSTOM_RETENTION_ENABLED" would have been a better choice for it?